<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MYCollaborativeEditWidget</class>
 <widget class="QWidget" name="MYCollaborativeEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MYCollaborativeEditWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="MYCollaborativeEditRoomWidget" name="widget_CERoom" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>402</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widget_CEGroup" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="MYCommitDataWidget" name="widget_Commit" native="true"/>
         <widget class="MYUpdateDataWidget" name="widget_Update" native="true"/>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Log">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MYCollaborativeEditRoomWidget</class>
   <extends>QWidget</extends>
   <header>MYCollaborativeEditRoomWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MYCommitDataWidget</class>
   <extends>QWidget</extends>
   <header>MYCommitDataWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MYUpdateDataWidget</class>
   <extends>QWidget</extends>
   <header>MYUpdateDataWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
